addons/l10n_no/data/account_tax_report_data.xml

```diff
@@ -1,291 +1,507 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
+    <!-- New tax report for Norway (January 2022)
+         Post in old version now have codes related to the SAF-T codes
+         Record ids are inaccurate due to complete changes in the tax report lines  -->
```

Comment:
I'm talking about the xml ids of the record, and the fact that they don't really represent what the the tax line is about anymore.
For instance, the first xml id is `id="account_tax_report_line_samlet_omsetning"` which means 'total turnover'. But the line itself is now about 'Sales of goods and services in Norway'

I didn't change the ids because we are in stable but it should be done in master